### ISSUE TITLE
Documentation release error

### ISSUE THREAD

**Issue by Keivan-sf** [alitnk/monopay#66]
While trying to merge #65, I faced build errors due to `@mdx-js/react` being v2 without having migration changes. I simply downgraded to v1 for now so the build could succeed. But after merging deploying to github pages failed with this error:
```
Error: Running "git push" command failed. Does the GitHub user account you are using have push access to the repository?
```
Seems to me this is the problem:
https://github.com/alitnk/monopay/blob/419a4f4a93e02a5c498aae05544fa95deb06dc80/.github/workflows/gh-page.yml#L43-L45

You have any context on this? And can you try to trigger the action with your account to see if it deploys? @alitnk

**Comment by cerpinn** [comment]
@Keivan-sf Thanks for looking into this ❤️ 

We keep the documentation website in the `main` branch now, under `apps/site`. I should have removed the old docs branch. I'll do that in a sec.

**Comment by cerpinn** [comment]
ba8881ba08c1524f549ee301c34acd442b5ec8af Should fix this issue and #64
